packages/zone/src/virtual.js

```diff
@@ -45,7 +45,7 @@ const makeVirtualExo = (
 };
 
 /** @type {import('.').Stores} */
-export const detachedVirtualStores = harden({
+const detachedVirtualStores = Far('virtualStores', {
```

Comment:
Removed the `export` because `detachedHeapStores` is not exported, and I do not see any reason it should be.